.git-blame-ignore-revs

```diff
@@ -72,3 +72,6 @@ e06d83860d798b6766b23d6eae48557387a7db85
 
 # Style: Replace header guards with `#pragma once`
 324512e11c1b7663c3cf47bec6ddbe65c6b8db2b
+
+# Style: Add scalar constructor to `Vector` types
+713cacfa7aa66d5a47bbd77b85c07e8df5d9d6e8
```

Comment:
This should not be blame ignored, it's a functional change, even if it's a minor change visually it's still a functional change, we should only ever ignore codestyle changes